numpy/core/multiarray.py

```diff
@@ -251,6 +251,21 @@ def concatenate(arrays, axis=None, out=None, *, dtype=None, casting=None):
     return arrays
 
 
+def concat(arrays, axis=None, out=None, *, dtype=None, casting=None):
+    """
+    concat((a1, a2, ...), axis=0, out=None, dtype=None, casting="same_kind")
+
+    Join a sequence of arrays along an existing axis.
+
+    `concat` is an alias of `~numpy.concatenate`.
+
+    See Also
+    --------
+    concatenate : alias of this function 
+    """
+    return concatenate(arrays, axis=None, out=None, dtype=None, casting=None)
+
```

Comment:
```
(numpy) D:\Open source cn\naampi\project\numpy>python runtests.py
Building, see build.log...
Traceback (most recent call last):
  File "D:\Open source cn\naampi\project\numpy\setup.py", line 45, in <module>
    raise RuntimeError(f'Cannot parse version {FULLVERSION}')
RuntimeError: Cannot parse version 0+untagged.31867.ga23209b

Build failed!
```

@MatteoRaso Sir, whenever I am trying to run the `python runtests.py`  file, I am getting this error. I have installed all the dependencies. Do I need to update the version number?

Reply:
> ```
> (numpy) D:\Open source cn\naampi\project\numpy>python runtests.py
> Building, see build.log...
> Traceback (most recent call last):
>   File "D:\Open source cn\naampi\project\numpy\setup.py", line 45, in <module>
>     raise RuntimeError(f'Cannot parse version {FULLVERSION}')
> RuntimeError: Cannot parse version 0+untagged.31867.ga23209b
> 
> Build failed!
> ```
> 
> @MatteoRaso Sir, whenever I am trying to run the `python runtests.py` file, I am getting this error. I have installed all the dependencies. Do I need to update the version number?

This might help: https://github.com/numpy/numpy/issues/21550